features/User/Controllers/UserController.cs

```diff
@@ -12,10 +12,27 @@ public UserController(IUserService userService)
         _userService = userService;
     }
 
-    [HttpGet]
+    [HttpGet("api/users")]
     public async Task<ActionResult<IEnumerable<User>>> GetAllUsers()
     {
         var users = await _userService.GetAllUsersAsync();
         return Ok(users);
     }
+
+    [HttpGet("api/users/{userID}")]
+    public async Task<ActionResult<User>> GetUser(String userID)
+    {
+        Guid? guid = null;
+
+        try{
+            guid = Guid.Parse(userID);
+        } catch (Exception){};
+
+        if(guid is not null){
+            var user = await _userService.GetUserAsync(guid.GetValueOrDefault());
```

Comment:
cia neturetu default value grazint jeigu neras guid, nes jau patikrinta, kad not null